.github/workflows/OCV-DNN-models-update.yaml

```diff
@@ -0,0 +1,60 @@
+name: OCV DNN Models Update
+
+on:
+  pull_request:
+    branches:
+      - main
+    paths:
+      - '.github/workflows/OCV-DNN-models-update.yaml'
+  schedule:
+    - cron: '0 0 * * *'
```

Comment:
It makes sense to trigger such updates 1 hour (30mins) before other nightly builds.

Reply:
Changed to `0 23 * * *`